# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mnemoc/ucspi-unix/head-useage.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2004-09-07 05:16:38.584125576 +0200
+++ ./Makefile	2004-09-07 05:17:05.934967616 +0200
@@ -18,18 +18,18 @@
 	( echo '#!/bin/sh'; \
 	  echo 'source=$$1; shift'; \
 	  echo 'base=`echo "$$source" | sed -e s:\\\\.c$$::`'; \
-	  echo exec `head -1 conf-cc` '-I. -o $${base}.o -c $$source $${1+"$$@"}'; \
+	  echo exec `head -n 1 conf-cc` '-I. -o $${base}.o -c $$source $${1+"$$@"}'; \
 	) >compile
 	chmod 755 compile
 
 conf_bin.c: conf-bin
-	head -1 conf-bin | \
+	head -n 1 conf-bin | \
 	  sed -e 's/"/\\"/g' \
 	      -e 's/^/const char conf_bin[] = "/' \
 	      -e 's/$$/";/' >conf_bin.c
 
 conf_man.c: conf-man
-	head -1 conf-man | \
+	head -n 1 conf-man | \
 	  sed -e 's/"/\\"/g' \
 	      -e 's/^/const char conf_man[] = "/' \
 	      -e 's/$$/";/' >conf_man.c
@@ -55,7 +55,7 @@
 load: conf-ld
 	( echo '#!/bin/sh';\
 	  echo 'main="$$1"; shift';\
-	  echo exec `head -1 conf-ld` '-o "$$main" "$$main.o" $${1+"$$@"}' -lsysdeps;\
+	  echo exec `head -n 1 conf-ld` '-o "$$main" "$$main.o" $${1+"$$@"}' -lsysdeps;\
 	) >load
 	chmod 755 load
 
